doc/mobile_base_arm/mobile_base_arm_tutorial.rst

```diff
@@ -0,0 +1,121 @@
+Planning For Differential-Drive Mobile Base and an Arm
+======================================================
+
+This tutorial will show how you can configure a mobile manipulator with a differential drive base so that you can plan coordinated motions for the base and the other degrees of freedom using MoveIt. While you could use MoveIt to plan for just the base, this is not recommended. Use Nav2 for pure navigation. Note that the tutorial uses Hello Robot's Stretch platform for demonstration purposes, the same approach can be used for any mobile manipulator with a diff-drive base.
+
+.. image:: images/intro.png
+   :width: 700px
+
+Getting Started
+---------------
+If you haven't already done so, make sure you've completed the steps in `Getting Started <../getting_started/getting_started.html>`_.
+
+Installation
+------------
+
+Install the dependencies: ::
+
+  cd ~/ws_moveit/src
+  wget https://raw.githubusercontent.com/PickNikRobotics/stretch_ros/ros2/stretch_ros.repos
```

Comment:
Do you think we should add the model to moveit resources?

Reply:
Yes, this's what we were planning to do, I'll create a follow-up PR to do so